lms/djangoapps/courseware/tests/test_masquerade.py

```diff
@@ -337,16 +340,22 @@ def test_masquerade_as_specific_user_on_self_paced(self):
         content = response.content
         self.assertIn("OOGIE BLOOGIE", content)
 
+    @ddt.data('john', 'foo@bar')
```

Comment:
`@ddt.data('john', 'foo@bar')` -> 

```
@ddt.data(
   'john', # Non-unicode username
   u'fôô@bar',  # Unicode username with @, which is what ENABLE_UNICODE_USERNAME feature allows
)
```

Reply:
Notice that I've added some non-ASCII Unicode chars here, to make sure it works. Better to test it than assume it'll work.